infra/recipes/terraform/azure/environments/sandbox/README.md

```diff
@@ -34,6 +34,20 @@ You could set the environment variables for variables to stored terraform backen
 `cp backend.tfvars.template backend.tfvars`.
 2. Edit the backend.tfvars to configure the necessary TF backend state.
 
+## Docker setup
```

Comment:
Just so I understand this, README flow should be organized in the order you want to user to execute the commands. After TF Backend setup do you want them to setup Docker or TF plan/apply for the backend?

If you need ACR provisioned, I would there are various ways in which we could organize the flow.

Option 1:
1. TF Backend setup for ACR
2. Docker image building for OH and publish to ACR
3. TF Backend setup for rest of infra (including deploying of services)

Option 2:
1. TF Backend setup for rest of infra (w/ ACR and w/o OH services deployment)
2. Docker image building for OH and publish to ACR
3. TF Backend setup only for OH services deployment.

Either is fine, but whichever we pick README should follow the flow clearly.